### DISCUSSION TITLE
Can i use lokinet in android?

### DISCUSSION THREAD

**Discussion by Zamhive** [oxen-io/lokinet#1963] [Q&A] [answered]
Any complete guidelines for android devices?

**Comment by majestrate** [comment, marked as answer]
we currently only have debug builds for q/a testing generated from our ci pipeline these are unreleased as they are not ready for normal users whatsoever as the ui is not finalized at all and many features remain unimplemented.

**Reply by majestrate** [reply]
> Does it actually set up lokinet though? 

yes
